style/main.css

```diff
@@ -0,0 +1,470 @@
+  font-weight: 600;
+  padding: 4px;
+  text-transform: uppercase;
+  float: left;
+}
+
+
+
+.imageSection__small {
+
+  position: relative;
+  float: left;
+  width: 334px;
+  height: 230px;
+  box-sizing: border-box;
+  overflow: hidden;
+  border: solid 5px rgb(255, 255, 255);
+  margin-right: 10px;
+  margin-bottom: 5px;
+}
+
+.imageSection__small--img1 {
+  display: block;
+  width: auto;
+  height: 100%;
+  left: 0;
+  float: right;
+
+}
+
+.imageSection__small--img2 {
+  display: block;
+  width: auto;
+  height: 100%
+  position: absolute;
+  right: 0;
+}
+.imageSection__medium {
+
+  position: relative;
+  float: left;
+  width: 676px;
+  height: 230px;
+  box-sizing: border-box;
+  border: solid 5px rgb(255, 255, 255);
+  margin-top: 10px;
+  margin-right: 10px;
+  margin-bottom: 5px;
+  overflow: hidden;
+}
+.imageSection__medium img {
+
+  width: auto;
+  height: 100%
+  position: absolute;
+  left: 50%;
+  top: 50%;
+
+  margin-top: -100px;
+}
+
+.imageSection__tall {
+  position: absolute;
+  top: 0;
+  right: 0;
+  float: right;
+  width: 333px;
+  height: 475px;
+  overflow: hidden;
+  border: solid 5px rgb(255, 255, 255);
+  margin-left: 10px;
+  margin-bottom: 5px;
+  box-sizing: border-box;
+}
+.imageSection__tall img {
+  display: block;
+  height: 100%;
+  width: auto;
+  float: right;
+  right: 50%;
+  top: 50%;
+  margin-right: -95px;
+
+}
+
+
+
+.imageSection__sk8 {
+  float: left;
+  clear: both;
+  background-color: rgb(255, 255, 255);
+  margin-top: 10px;
+
+  column-count:10;
+  -moz-column-count:10;
+  -webkit-column-count: 10;
+
+  column-gap:20px;
+  -moz-column-gap:20px;
+  -webkit-column-gap: 20px;
+
+  height: 295px
+  width: 1024px;
+  margin-bottom: 22px;
+
+}
+
+.pageEnd {
+  position: relative;
+  display: inline-block;
+  box-sizing: border-box;
+  width: 1024px;
+  height: 360px;
+  padding: 33px;
+  background-color: rgb(255, 255, 255);
+  border-bottom: 2px rgba(149, 165, 166, 0.5);
+  margin-top: 22px;
+  }
+
+.pageEnd__wrapper {
+  height: auto;
+  width: 490px;
+
+
+}
+.pageEnd__title {
+  display: inline-block;
+  font-size: 22px;
+  font-weight: 400;
+  margin-top: 0px;
+  margin-bottom: 15px;
+  }
+
+  .pageEnd__text{
+    display: inline-block;
+    font-size: 15px;
+    margin-bottom: 25px;
+    line-height: 150%;
+  }
+  .pageEnd__graySocialMediaLinks img {
+    display: inline;
+    height: 30px;
+    width: 30px;
+    margin-right: 3px;
+  }
+  /*.pageEnd__graySocialMediaLinks {
+    display: inline;
+    height: 30px;
+    width: 30px;
+    margin: 10px;
+  }*/
+
+
+  .submit {
+    position: absolute;
+    right: 0;
+    display: inline-block;
+    width: 246px;
+    height: 30px;
+    margin-right: 10px;
+    margin-bottom: 16px;
+    background-color: rgb(180, 41, 46);
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: rgb(255, 255, 255);
+    padding: 8px;
+    font-size: 16px;
+    font-weight: 500;
+    box-sizing: border-box;
```

Comment:
Have seen this throughout your styling, could move this to a selector like `*` to have all elements affected by this.